src/inject.rs

```diff
@@ -0,0 +1,66 @@
+extern crate parity_wasm;
```

Comment:
Do we still need this file? What does it do?

Reply:
You're right, let me remove that!